src/alfasim_sdk/_internal/types.py

```diff
@@ -667,6 +680,14 @@ class Table(BaseField):
 
     .. code-block:: python
 
+        @data_model(caption="Wolf")
+        class Wolf:
```

Comment:
This example feels a bit generic/silly in a SDK for a flow assurance simulator, perhaps use something else like "TensileProfile" and "TensileProfiles" (related to tube tensile strength)? Anything that sounds related to a simulation would be better I think.

Reply:
- [ ] use "alfasim" problem domain thematic names in example;

Reply:
btw tensile strength sounds like a property of something, not an entity by itself.

Reply:
The new rendered docs are at:

https://alfasim-sdk--413.org.readthedocs.build/en/413/plugins/99_2_user_interface_reference.html#alfasim_sdk.Table

Hit the `d` key to toggle between the plain rendered version and "diff" one.